api/client/ps/custom.go

```diff
@@ -65,7 +65,9 @@ func (c *containerContext) Image() string {
 		return "<no image>"
 	}
 	if c.trunc {
-		return stringutils.Truncate(c.c.Image, 12)
+		if len(c.c.Image) == 64 {
```

Comment:
Oh, I get it now. `== 64` doesn't sound very useful, what about `> 20`? Please keep using `stringsutils.Truncate`, it's more correct. `stringid.TruncateID` should actually use that rather than having its own implementation.